bears/python/BlackBear.py

```diff
@@ -0,0 +1,39 @@
+from coalib.bears.LocalBear import LocalBear
+from dependency_management.requirements.DistributionRequirement import DistributionRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class BlackBear(LocalBear):
+    LANGUAGES = {'Python', 'Python 2', 'Python 3'}
+    REQUIREMENTS = {
+        DistributionRequirement('python3', version='3.6'),
+        PipRequirement('black', '18.4a4'),
+    }
+    AUTHORS = {'Bence Nagy'}
+    AUTHORS_EMAILS = {'bence@underyx.me'}
```

Comment:
Look at the this metadata from other bears... it should be `The coala developers` :wink:
also change the email accordingly